code/game/objects/items/stacks/medical.dm

```diff
@@ -174,6 +176,7 @@
 	flags_atom = DIRLOCK
 	skill_level_needed = SKILL_MEDICAL_PRACTICED
 	unskilled_delay = SKILL_TASK_EASY
+	toolspeed = 8 SECONDS
```

Comment:
I think this has too much consequences on corps trying to defib someone. 

What about doing this BUT giving corp like a tool to heal dead bodies?

Reply:
Or making a device in medbay to heal bodies back to 200 maybe (a cryo change?)

Reply:
Wait is this seriously making kits even shittier than they already are?

Reply:
We have that, it's called sutures.

Reply:
changed to 3.5.